README.md

```diff
@@ -99,6 +99,16 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with
+
+   ```
+   # yay -S starship
```

Comment:
```suggestion
   $ yay -S starship
```

Reply:
So this is a fun one.

Conventionally in Arch, the `$` sign is used for regular user prompts, while the `#` sign is used for superuser prompts.

So we can do `# yay -S starship` or `$ sudo yay -S starship` but probably not `$` without sudo.

Reply:
Huh... Alright.
No worries 👍